app/src/main/java/io/github/droidkaigi/confsched/activity/ContributorsActivity.java

```diff
@@ -94,9 +93,21 @@ private void renderSavedContributors(Throwable throwable) {
         Crashlytics.logException(throwable);
 
         List<Contributor> contributors = dao.findAll();
-        ContributorsAdapter contributorsAdapter = new ContributorsAdapter(this);
-        contributorsAdapter.addAll(contributors);
-        binding.recyclerView.setAdapter(contributorsAdapter);
+
+        if (contributors.isEmpty()) {
+            Snackbar.make(binding.getRoot(), R.string.contributors_load_error, Snackbar.LENGTH_LONG).show();
+        } else {
+            bindContributors(contributors);
+        }
+    }
+
+    private void bindContributors(List<Contributor> contributors) {
+        String title = binding.toolbar.getTitle().toString();
+        binding.toolbar.setTitle(title + getString(R.string.number_with_brackets, contributors.size()));
```

Comment:
Add number of contributors to title.

<img width="398" alt="2016-02-14_02_12_44_png" src="https://cloud.githubusercontent.com/assets/1269214/13031298/0b6e3580-d30c-11e5-8618-489597551ef7.png">